Jorge Review to Languages, and more

jorgedlcruz opened the pull request:
Here are all the changes I made for the Install Guides, system requirements, delete the Suse which is deprecated for so long, Languages, and much more.

jorgedlcruz committed:
Expanded Ports section

Expanded the Ports section using the next wiki -
https://wiki.zimbra.com/wiki/Ports

jorgedlcruz committed:
Updated OS

Updated the OS list, and removed Use, not supported for long time

jorgedlcruz committed:
minor change on OS

minor change on OS

jorgedlcruz committed:
removed SLES

removed SLES

jorgedlcruz committed:
Added Zimbra Chat and Drive

Added Zimbra Chat and Drive

jorgedlcruz committed:
minor edits

jorgedlcruz committed:
Adding Chat and Drive

Adding Chat and Drive

jorgedlcruz committed:
Added Zimbra Chat and 8.8

Added Zimbra Chat and 8.8, added some extra comments for dis-cache as
well.

jorgedlcruz committed:
Languages and more

Languages and more

jorgedlcruz committed:
minor change

jorgedlcruz committed:
minor edit

jorgedlcruz committed:
final languages

tmclane commented on installing.adoc:
```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
-so it can install the {product-short} 3rd party packages.
+so it can install the {product-short} 3rd party packages. 
+*{product-short} IMAPD is meant to be installed on a multi-server environment, with
+hundred of thousands of users, and it's still as Beta. {product-short} recommends
```
as Beta = a Beta release

jorgedlcruz replied:
changed and added a new version

jorgedlcruz committed:
minor edit

jorgedlcruz committed:
minor edit on xmpp

gordyt commented:
@jorgedlcruz  I am verifying this with John but I think there may be some additional updates required, similar to the work that John already did here:

    bf714a6 [John Eastman] Cleanup in preparation for translations.
    Ordered lists have been indented per recommendation from po4a. 
    Indentation cleanup of existing lists.

Will let you know what I find out.

jorgedlcruz commented:
Okay @gordyt please let me know, the edits were so extensive across the whole document. We are aiming to have this as /latest as discussed before. Let me know the edits we should add as well. Or maybe we can merge this and then do the additional changes after.

gordyt commented:
Will do.  I believe that we will want to make these changes first.  If I understand the issue correctly, there are some problems generating the files required to do language translation without them.  

BTW, is OK for now, but for future PRs please base them off of the `develop` branch.  Note that I did do the whole `latest` thing yesterday (for admin and install guides), and is very easy to update after we merge your PR.

gordyt commented on installing.adoc:
```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
 the {product-short} packaging repository for *yum* or *apt-get* as appropriate
-so it can install the {product-short} 3rd party packages.
+so it can install the {product-short} 3rd party packages. 
+*{product-short} IMAPD is meant to be installed on a multi-server environment, with
```
@jorgedlcruz Here and anywhere else where you have bulleted or numbered list items, please be sure to have a single space between the marker and the text that follows.

jorgedlcruz replied:
fixed, it didn't need bold or bullet

gordyt commented on packages.adoc:
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
++
+
+* *{product-short} Drive:* Installing this package is optional.This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Drive includes a server extension with all 
```
same here

jorgedlcruz replied:
fixed, it didn't need bold

gordyt commented on packages.adoc:
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
+the necessary modules to connect and authenticate *{product-short} Users against a ownCloud or
+NextCloud Server, and an End-User Zimlet which allow users to perform actions to the their docu-
+ments stored on ownCloud or Nextcloud.
+*{product-short} Drive is marked as GA and supported starting {product-name} 8.8.
```
ditto

jorgedlcruz replied:
fixed, it didn't need bold

gordyt commented on systemrequirements.adoc:
```diff
@@ -115,31 +116,26 @@ a|Administrator Computers +
-// TODO this looks well out of date.  How about Windows 10?  Edge browser?
-Windows XP with required updates, Vista, Windows 7, or Windows 8 with
-one of the following:
+*Windows Vista SP 2, Windows 7, Windows 8 or Windows 10 with one of the following:*
```
here

jorgedlcruz replied:
This is bold not bullet

gordyt commented on systemrequirements.adoc:
```diff
@@ -115,31 +116,26 @@ a|Administrator Computers +
+** IE11 or Microsoft Edge (Supported since ZCS 8.6 P4 and above) for Windows 10 
 
-* The latest stable release of:
+*The latest stable release of:
```
here

jorgedlcruz replied:
fixed

gordyt commented on systemrequirements.adoc:
```diff
@@ -162,31 +158,26 @@ Recommended
 The following operating system/browser combinations are supported:
 
-Windows XP with required updates, Vista, Windows 7, Windows 8 or Windows 10 with one of the following:
+*Windows Vista SP 2, Windows 7, Windows 8 or Windows 10 with one of the following:*
```
here

jorgedlcruz replied:
This is bold not bullet

gordyt commented on systemrequirements.adoc:
```diff
@@ -162,31 +158,26 @@ Recommended
-Mac OS X 10.5, 10.6, 10.7, or 10.8 with one of the following:
-
-* The latest stable release of:
+*The latest stable release of:
```
here

jorgedlcruz replied:
fixed

gordyt reviewed (commented):
@jorgedlcruz I've noted the places that need that minor update for compatibility with the translation tools.

gordyt commented:
@jorgedlcruz let me go ahead and generate a test export so folks can see the final product.
One sec...

jorgedlcruz committed:
minor edits

jorgedlcruz commented:
Appreciated @gordyt thank you as usual for all the hard work!

gordyt committed:
Fixed list issue:

$ asciidoctor -b html5 -o single.html index-single-server-install.adoc

asciidoctor: WARNING: installing.adoc: line 330: list item index: expected 15, got 16
asciidoctor: WARNING: installing.adoc: line 338: list item index: expected 16, got 17
asciidoctor: WARNING: installing.adoc: line 349: list item index: expected 17, got 18

gordyt commented:
@jorgedlcruz  I had to push in one fix to your branch for this:

    $ asciidoctor -b html5 -o single.html index-single-server-install.adoc
    asciidoctor: WARNING: installing.adoc: line 330: list item index: expected 15, got 16
    asciidoctor: WARNING: installing.adoc: line 338: list item index: expected 16, got 17
    asciidoctor: WARNING: installing.adoc: line 349: list item index: expected 17, got 18

Generated a `gh-pages` export with the results.  You can view them here:

* [single-server](https://zimbra.github.io/installguides/work-in-progress/single.html)
* [multi-server](https://zimbra.github.io/installguides/work-in-progress/multi.html)
* [upgrade](https://zimbra.github.io/installguides/work-in-progress/upgrade.html)

Please go through the results and check the formatting.

plobbes commented on installing.adoc:
```diff
@@ -96,59 +103,70 @@ Found zimbra-store (local)
+Install zimbra-drive [Y] y
 
-Install zimbra-drive [Y] N
+Install zimbra-imapd [Y] n            
```
nit: trailing whitespace

gordyt commented:
@jorgedlcruz I'm adding @gaylebillat as a content reviewer.

plobbes commented on systemrequirements.adoc:
```diff
@@ -115,31 +116,26 @@ a|Administrator Computers +
 * The latest stable release of:
 ** Firefox
-** Safari
+** Safari 
```
nit: trailing whitespace

jorgedlcruz replied:
done

plobbes commented on systemrequirements.adoc:
```diff
@@ -162,31 +158,26 @@ Recommended
 * The latest stable release of:
 ** Firefox
-** Safari
+** Safari 
```
nit: trailing whitespace

jorgedlcruz replied:
done

plobbes reviewed (commented):
whitespace nits

jorgedlcruz committed:
minor edits

jorgedlcruz committed:
Merge remote-tracking branch 'origin/jorge-review' into jorge-review

# Conflicts:
#	installing.adoc

fatkudu commented on installconfig.adoc:
```diff
@@ -79,6 +79,17 @@ endif::networkeditiondoc[]
+  be installed on every Zimbra Store Server that is part of the Zimbra configuration.
+
+| zimbra-drive
+| Installing the Zimbra-Chat package is optional.
```
s/Zimbra-Chat/Zimbra/Drive/

jorgedlcruz replied:
done

fatkudu commented on installconfig.adoc:
```diff
@@ -79,6 +79,17 @@ endif::networkeditiondoc[]
+
+| zimbra-drive
+| Installing the Zimbra-Chat package is optional.
+  If you choose to install Zimbra-Chat for point-to-point text chat, it should
```
Is this true for Zimbra-Drive true?  If so, update accordingly or replace with appropriate text.

jorgedlcruz replied:
done

fatkudu commented on installing.adoc:
```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
-so it can install the {product-short} 3rd party packages.
+so it can install the {product-short} 3rd party packages. 
+{product-short} IMAPD is meant to be installed on a multi-server environment, with
+hundred of thousands of users, and it's still a Beta Release. {product-short} recommends
```
s/hundred/hundreds/

jorgedlcruz replied:
done

fatkudu commented on installing.adoc:
```diff
@@ -79,7 +79,14 @@ and you must accept each section of the license agreement.
+to only install it for demo and evaluation purposes.*
+* If installing *{product-short} Network Edition*, you will have the modules Archiving
+and Convertd as an option to add to your {product-short} installation.
+* {product-short}-dnscache is recommended to be installed on the MTA Servers only when
```
It is recommended to only install *{product-short}-dnscache* on the MTA Servers  when there are no other DNS servers (for example *dnsmasq* or *bind*) already installed and running on the same server.

jorgedlcruz replied:
done

fatkudu commented on installing.adoc:
```diff
@@ -96,59 +103,70 @@ Found zimbra-store (local)
+
+Network Modules NG needs to bind on TCP ports 8735 and 8736 in order
+to operate, for inter-instance communication.
+Please verify no other service listens on these ports and that 
```
Please verify that no other service...

jorgedlcruz replied:
This is a comment on the own product installation, I will not edit it on the documentation.

fatkudu commented on packages.adoc:
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
 endif::networkeditiondoc[]
 
+* *{product-short} Chat:* Installing this package is optional. This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Chat includes a server extension with all 
```
Missing '*' after Chat

jorgedlcruz replied:
done

fatkudu commented on packages.adoc:
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
+{product-short} Chat is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Chat package must be installed on the different *{product-short}-Store* server
```
I don't understand the meaning of this sentence - sorry.

jorgedlcruz replied:
edited

fatkudu commented on packages.adoc:
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
+{product-short} Drive is marked as GA and supported starting {product-name} 8.8.
++
+[NOTE]
+The {product-short} Drive package must be installed on the different *{product-short}-Store* server
```
Again, don't actually know what this means.  Does Drive need to be on its own server?  How does that work?

jorgedlcruz replied:
The Drive and Chat package for Zimbra consist on a server extension and a zimlet, so the Drive or Chat package must be installed on all Zimbra-Stores on the Infrastructure. Or at least on the ones where we want the users to be able to access Drive/Chat. I've edited, take a look now.

fatkudu commented on packages.adoc:
```diff
@@ -109,6 +109,34 @@ license usage. To find out more about {product-short} Archiving and Discovery,
+
+[WARNING]
+Zimbra Drive provides only a connectivity to a ownCloud or NextCloud Server. And is the Customer
+responsibility to maintain, backup, and protect the data stored on this OC or NC Servers.
```
s/on this OC or NC Servers/on these ownCloud or NextCloud Servers/

jorgedlcruz replied:
done

jorgedlcruz committed:
minor edits

jorgedlcruz committed:
minor edits

jorgedlcruz committed:
minor edits

jorgedlcruz committed:
minor edits

jorgedlcruz committed:
minor edits

jorgedlcruz committed:
minor edits

fatkudu commented on packages.adoc:
```diff
@@ -109,6 +109,32 @@ license usage. To find out more about {product-short} Archiving and Discovery,
++
+
+* *{product-short} Drive:* Installing this package is optional.This package should be installed on at least one
+*{product-short}-Store* server. *{product-short} Drive includes a server extension with all 
```
Looks like a couple of places here and on the next line where BOLD is started but not finished?

fatkudu reviewed (changes requested):
Looking good.  Only 1 minor query about bolding.

jorgedlcruz commented:
So now with all the suggested changes, @gordyt do you think you can generate another HTML, which I hope is the final version, for now at least?

gordyt commented:
@jorgedlcruz will do.  Gotta finish a few things first.  Will post update here when live.

gordyt commented:
@jorgedlcruz I have redone the export and went ahead and published it to the `8.8.3` (`latest`) location.

* [single-server](https://zimbra.github.io/installguides/latest/single.html)
* [multi-server](https://zimbra.github.io/installguides/latest/multi.html)
* [upgrade](https://zimbra.github.io/installguides/latest/upgrade.html)

*If* you do have to make any additional changes, please be sure and update your local `jorge-review` branch with the latest copy of it from `origin`.  You had inadvertently done a merge from `origin/jorge-review` to your local `jorge-review` (rather than rebasing your local `jorge-review` onto `origin/jorge-review`) and ended up with a wonky merge commit. 

For example, you would do the following from your local checkout:

    git fetch origin
    git checkout -B jorge-review origin/jorge-review

That will reset your *local* branch to be correctly in sync with `origin`.

jorgedlcruz committed:
Merge remote-tracking branch 'origin/jorge-review' into jorge-review
